docs/security-and-authentication.rst

```diff
@@ -34,3 +34,50 @@ local Runhouse config (:code:`~/.rh/config.yaml`), or in Python:
 
     import runhouse as rh
     rh.configs.disable_data_collection()
+
+
+Cluster Observability
+---------------------------------------
+Runhouse collects various telemetry data by default on clusters. This data will be used to provide better observability
+into logs, traces, and metrics associated with clusters. We will not sell data or buy any observability data collected.
+
+To disable observability globally for all clusters, set the environment variable :code:`disable_observability`
+to :code:`True`. Alternatively, set :code:`disable_observability` to :code:`true` in your
+local Runhouse config (:code:`~/.rh/config.yaml`), or in Python:
+
+.. code-block:: python
+
+    import runhouse as rh
+    rh.configs.disable_observability()
+
+Please note that the config.yaml and the disable_observability environment variable are **global settings**, which apply to every new cluster you will create using Runhouse.
+
+To disable observability for a specific cluster, you may restart the cluster Runhouse server, using `Runhouse Cluster API <https://www.run.house/docs/api/python/cluster#runhouse.Cluster.restart_server>`__.
+
+.. code-block:: python
+
+    import runhouse as rh
+    my_cluster = rh.cluster(name="my-gpu-cluster", instance="CPU:2+").up_if_not()
+    my_cluster.restart_server(disable_observability=True)  # for disabling observability on the specific cluster
```

Comment:
not sure if this is the usage path we want to highlight for disabling observability, bc in this case its entirely possible that the cluster will be started with it enabled